pandas/tests/plotting/test_misc.py

```diff
@@ -77,6 +77,12 @@ def scat(**kwds):
             _check_plot_works(scat, diagonal='hist')
         with tm.assert_produces_warning(UserWarning):
             _check_plot_works(scat, range_padding=.1)
+        with tm.assert_produces_warning(UserWarning):
+            _check_plot_works(scat, color='rgb')
```

Comment:
Is there a reason you pass here 'rgb' and not just 'r'? As passing multiple colors seems a bit odd for a single scatter plot (I think now the points just get the [r, g, b, r, g, b, r, g, b, ...] colors sequentially?)